worlds/messenger/__init__.py

```diff
@@ -94,6 +97,20 @@ def create_items(self) -> None:
                          # if we get in a position where this can have duplicates of items that aren't Power Seals
                          # or Time shards, this will need to be redone.
                      }]
+
+        if self.multiworld.goal[self.player] == Goal.option_power_seal_hunt:
```

Comment:
Can't be both
```suggestion
        elif self.multiworld.goal[self.player] == Goal.option_power_seal_hunt:
```

Reply:
There's stuff in between the 2 if comparisons.

Reply:
Yeah, I meant it could be reordered for the elif so it is two distinct code paths side by side. But if you really think it's nicer this way, we can leave it.

Reply:
It's uglier this way, I agree. The reason I do it this way is so that the amount of power seals can potentially vary based on starting inventory.